Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM python:3.8-slim
+FROM python:3.8-slim-bullseye
```

Comment:
I understand the reason for switching to bullseye is to make `netcat` work/install properly.  This is only needed on the OpenShift 3.11 cluster, but not ROSA.  In fact, on ROSA (if this project gets migrated there), we may need to revert this change to `bullseye`.

If the change is primarily because `netcat` won't install/work properly, maybe there's another solution.  I see `netcat` is only used in the local startup script to test whether the DB is available.  I think `netcat` doesn't even need to be installed when running on OpenShift.  Is there some other tool that could be used for this?  Perhaps `socat`?

Also, I've read that changing the `netcat` dependency to `netcat-traditional` will make that tool work under `bookworm`.

Reply:
We _probably_ should have had bullseye in there from the start for a more predictable build but we haven't seen a major switch like this before. That was until Python recently switched all of their default (non explicit) tags over to a more recent base OS. (bookworm)

Since we've already been using bullseye it seemed less risky to just keep everything on bullseye for now rather than upgrading the OS version. That _could_ introduce other issues other than just some of these dependencies not working at startup. Then next release (if there is one) upgrading and testing the apps to bookworm.

We could use either `netcat-traditional`, `netcat-openbsd` or possibly something else entirely to check the MySQL status. We could _probably_ have a small python program that just uses `socket `and tries to connect and check if it's running, since Python is guaranteed to be available.

Reply:
This project does need to get migrated to ROSA, since it's going to be running past August. I don't think this change would ever need to be reverted in ROSA, it should work fine with the `bullseye` image.

You are correct that `netcat` is just used for just local development; I don't think an alternative is needed in deployments. It would be better to layer or remove this from the final image to keep them smaller. That's not in scope for this issue, and it's not a priority for this project.

Let me know what you want me to do here.